### PR TITLE
enable link issue in navbar solved

### PR DESCRIPTION
**Pull request by Garvit-32**

### CODE REVIEW

File: templates/main/base.html [hunk 1]
```diff
@@ -70,14 +70,8 @@
...
                                     </a>
-                                    <a href="#"
-                                       onclick="alert('varchas is postponed new dates will be anounced soon.')">
-                                        <li class="topbar-styles">Leave Team</li>
-                                    </a>
-                                    <a href="#"
-                                       onclick="alert('varchas is postponed new dates will be anounced soon.')">
-                                        <li class="topbar-styles">Remove player</li>
-                                    </a>
+                                    <a href="{% url 'accounts:leaveTeam' %}"><li>Leave Team</li></a>
```
Comment (killbotXD) [inline templates/main/base.html:73]: why did you remove the style class?

File: templates/main/base.html [hunk 2]
```diff
@@ -89,14 +83,8 @@
...
                                 <ul>
-                                    <a href="#"
-                                       onclick="alert('varchas is postponed new dates will be anounced soon.')">
-                                        <li class="topbar-styles">Join Team</li>
-                                    </a>
-                                    <a href="#"
-                                       onclick="alert('varchas is postponed new dates will be anounced soon.')">
-                                        <li class="topbar-styles">Create Team</li>
-                                    </a>
+                                    <a href="{% url 'accounts:joinTeam' %}"><li>Join Team</li></a>
```
Comment (killbotXD) [inline templates/main/base.html:86]: same here

File: templates/main/base.html [hunk 3]
```diff
@@ -116,14 +104,8 @@
...
-                                {#                                <a href="{% url 'accounts:register' %}"><li>Register as Player</li></a>#}
-                                {#                                <a href="{% url 'registration:ca' %}"><li>Campus Ambassador</li></a>#}
-                                <a onclick="alert('varchas is postponed new dates will be anounced soon.')">
-                                    <li class="topbar-styles">Register as Player</li>
-                                </a>
-                                <a onclick="alert('varchas is postponed new dates will be anounced soon.')">
-                                    <li class="topbar-styles">Campus Ambassador</li>
-                                </a>
+                                <a href="{% url 'accounts:register' %}"><li>Register as Player</li></a>
+                                <a href="{% url 'registration:ca' %}"><li>Campus Ambassador</li></a>
```
Comment (killbotXD) [inline templates/main/base.html:108]: same here

File: templates/main/base.html (hunk 1 above)
Comment (Garvit-32) [reply templates/main/base.html:73]: it was by mistake
